### PR TITLE
[Issue #103] Updated scheduling availability page

### PR DESCRIPTION
**Pull request by mohameddahrouj**
Only one button

### CODE REVIEW

File: src/main/resources/static/views/projects/schedule.html [hunk 1]
```diff
@@ -382,11 +378,12 @@ <h4 class="panel-title">
                                         </div>
                                     </li>
                                 </ul>
-                                <button type="button" class="btn btn-success" style="float: right;">Submit Availability</button>
                             </div>
                         </div>
                     </div>
                 </div>
+                <br>
+                <button type="button" class="btn btn-success" style="float: right;">Submit Availability</button>
```
Comment (loushiano) [inline src/main/resources/static/views/projects/schedule.html]: is there a way i can put all the information in a model like class, such as the email one we did?

Comment (mohameddahrouj) [reply src/main/resources/static/views/projects/schedule.html]: Yes there is, Tosin was saying there is a way.

Comment (loushiano) [reply src/main/resources/static/views/projects/schedule.html]: okay perfect, i will be waiting until that is done to work on the integration part and test it on my end.

Comment (tosinoni) [reply src/main/resources/static/views/projects/schedule.html]: Hmm my part cannot be done unless i see your changes. I need to know how the scheduleDTO looks like, so i can rewrite the html to fit that.

Comment (mohameddahrouj) [reply src/main/resources/static/views/projects/schedule.html]: lets finish submissions, then move onto scheduling

**Pull request merged**